install_pim/customize_dataset/index.rst

```diff
@@ -113,3 +113,21 @@ You can now (re)install your database by running:
 .. code-block:: bash
 
     php bin/console pim:installer:db --env=prod
+
+What we couldn't have in the Dataset
```

Comment:
maybe write: "what you cannot have in the dataset"
Couldn't implies that we tried but we were not able to do it.